pom.xml

```diff
@@ -18,6 +18,10 @@
             <id>public.onebusaway.org</id>
             <url>http://nexus.onebusaway.org/content/groups/public/</url>
         </repository>
+        <repository>
```

Comment:
@barbeau that repository was there before this commit and contains artifacts needed for 
<!-- Core GTFS Library -->
        <dependency>
            <groupId>org.onebusaway</groupId>
            <artifactId>onebusaway-gtfs</artifactId>
            <version>1.3.4</version>
        </dependency>

        <!-- Optional Hibernate GTFS Database Persistence -->
        <dependency>
            <groupId>org.onebusaway</groupId>
            <artifactId>onebusaway-gtfs-hibernate</artifactId>
            <version>1.3.4</version>
            <exclusions>
                <exclusion>
                    <groupId>org.apache.commons</groupId>
                    <artifactId>collections</artifactId>
                </exclusion>
            </exclusions>
        </dependency>

        <dependency>
            <groupId>org.onebusaway</groupId>
            <artifactId>onebusaway-gtfs-realtime-exporter</artifactId>
            <version>1.0.1</version>
        </dependency>